### DISCUSSION TITLE
'iostream' file not found in macos after update.

### DISCUSSION THREAD

**Discussion by ShuKhi22** [clangd/clangd#2197] [Q&A]
'iostream' file not found in macos after update. When i check the path of include there is not iostream there

**Comment by OS-Scratch** [comment]
Same on Windows! using the newest VSCode and I can't code in C++ now.

**Comment by HighCommander4** [comment]
Please have a look at https://clangd.llvm.org/troubleshooting#cant-find-standard-library-headers-map-stdioh-etc

**Comment by OS-Scratch** [comment]
OK! But how can I find out the path of the file?

**Reply by HighCommander4** [reply]
What file?
